src/js/view/view.js

```diff
@@ -417,12 +417,15 @@ define([
 
         function itemReady(item) {
             const provider = _model.getVideo();
-            if (provider && provider.getName().name.indexOf('flash') === 0) {
+            // Youtube, chromecast and flash providers do no support video tags
+            if (provider && /^(youtube|chromecast|flash)/.test(provider.getName().name)) {
                 return;
             }
-            const title = item.title || '';
             var videotag = _videoLayer.querySelector('video, audio');
-            videotag.setAttribute('title', title);
+            const dummyDiv = document.createElement('DIV');
```

Comment:
@robwalch I'll exclude it for now, and bring it up for Today's tech debt meeting. There's probably more dummy divs across the player, it's worth seeing how feasible it is to change those to all use the same static element used by all players, but not sure how risky this is